core/modules/url_filter.h

```diff
@@ -21,6 +24,9 @@ class Flow {
   uint32_t dst_ip;
   uint16_t src_port;
   uint16_t dst_port;
+  uint32_t padding;
+
+  Flow() : padding(0) {}
```

Comment:
If I am not misunderstanding, you were making two different points here:
1. Remove the constructor because members should get zero-initialized: I think Sangjin is right that this is not true, so it is necessary to initialize them in ctor.
2. Initialize all members consistently in the ctor: This is fine to me. Also, this contradicts with the first point -- just to point it out.